apps/docs/getting-started/monorepo-setup/bun.mdx

```diff
@@ -24,7 +24,7 @@ Include a new `package.json` and do not forget to add this to the `workspaces` o
 Install React Email to the `transactional` workspace.
 
 ```sh packages/transactional
-bun add react-email @react-email/components -E
```

Comment:
Noticed that `react-email` is also not being installed as a devDependency, can you take care of that as well?

Reply:
Sure!